### PR TITLE
Select2 4.0.13 migration

### PR DESCRIPTION
**Pull request by joehybird**

### CODE REVIEW

**Comment by hsmett** [comment]
Any change required for the other theme ?

File: CHANGELOG.txt [hunk 1]
```diff
@@ -51,6 +52,8 @@
         - New dependency 'underscorejs 1.13.2'
         - The version of 'floathead' is now "2.2.4".
         - The version of 'jquery-ui' is now "1.13.1".
+    - Chose to Select2 migration :
```
Comment (hsmett) [inline CHANGELOG.txt:55]: Chosen

File: creme/creme_core/static/creme_core/js/widgets/form/select2.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+creme.form = creme.form || {};
+
+creme.form.Select2 = creme.component.Component.sub({
+    _init_: function(options) {
+        this._options = $.extend({
+            multiple: false,
+            sortable: false,
+            clearable: false,
+            noResults: gettext("No result"),
```
Comment (genglert) [inline creme/creme_core/static/creme_core/js/widgets/form/select2.js:30]: Refresh djangojs.po

File: creme/creme_core/templates/creme_core/forms/widgets/datetime.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {% load i18n %}
 {% with attrs=widget.attrs %}
```
Comment (genglert) [inline creme/creme_core/templates/creme_core/forms/widgets/datetime.html:2]: Useless now?

File: creme/settings.py [hunk 4]
```diff
@@ -896,6 +897,9 @@
 
     'creme_core/js/widgets/list/pager.js',
 
+    'creme_core/js/widgets/form/chosen.js',
```
Comment (genglert) [inline creme/settings.py:900]: Can now add a comment like "# DEPRECATED: to be removed in Creme 2.5" please, so I won't forget it.

File: CHANGELOG.txt [hunk 5]
```diff
@@ -214,6 +217,8 @@
         # In JavaScript :
             - The function 'creme.forms._toDualColumnMultiSelect' & 'creme.forms.toOrderedMultiSelect' have been removed ;
               the new component 'creme.widget.SelectOrInputWidget' is used instead.
+            - 'creme.component.Chosen' has been renamed as 'creme.form.Chosen' and moved into 'widget/form/chosen.js' file.
+            - 'creme.form.Chosen' is now deprecated : no longer used in media generation but still exist to ease migration.
```
Comment (genglert) [inline CHANGELOG.txt:221]: It's only one item : "creme.component.Chosen is now deprecated ; but it has been renamed ..."

**Review by genglert — CHANGES_REQUESTED** [review]

File: CHANGELOG.txt (hunk 1 above)
Comment (joehybird) [reply CHANGELOG.txt:55]: :sweat_smile:

File: CHANGELOG.txt (hunk 5 above)
Comment (joehybird) [reply CHANGELOG.txt:221]: ok

File: creme/creme_core/locale/fr/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Creme Creme-Core 2.4\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2022-06-29 21:40+0200\n"
+"POT-Creation-Date: 2022-07-04 12:25+0100\n"
```
Comment (genglert) [inline creme/creme_core/locale/fr/LC_MESSAGES/django.po:11]: Revert

**Review by genglert — CHANGES_REQUESTED** [review]

**Pull request merged**